README.md

```diff
@@ -33,6 +33,15 @@ generated output.
 Including files manually like this lets us specify the position they appear in
 the page.
 
+## Raising and merging PRs to this repo
+
+To submit changes to this repo, raise a PR in the usual way and these will be regularly reviewed by The GDS Way forum group that meets once a month.
```

Comment:
we should make it clear that merging PRs outside of a forum meeting is fine, they'll still get reviewed at forum

Reply:
it still looks like you raise a PR in the usual way and the forum will merge?